using-mobile-extensions/adobe-audience-manager/README.md

```diff
@@ -22,15 +22,36 @@ To get started with **Audience Manager**, follow these steps:
 5. Click **Save**.
 6. Follow the publishing process to update SDK configuration.
 
-## Add Audience Manager to your App
-
-**Java**
+### Add Audience Manager to your App
 
+{% tabs %}
+{% tab title="Android" %}
 1. Add the library to your project.
 2. Import the library:
 
+#### Java
 `import com.adobe.marketing.mobile.*;`
 
+{% tab title="iOS" %}
+1. Add the library to your project via your `Podfile` by adding `pod 'ACPAudience'`
+2. Import the Audience and Identity library:
+
+#### Objective-C
+  ```objectivec
+  #import "ACPCore.h"
+  #import "ACPAudience.h"
+  #import "ACPIdentity.h"
+  ```
+
+#### Swift
+  ```swift
+   import ACPCore
+   import ACPAudience
+  ```
+
+   **Important**: Audience Manager depends on the Identity extension and is automatically included in the Core pod. When installing manually, ensure that you have also added the `ACPIdentity.a` to your project.
```

Comment:
I think we should replace `ACPIdentity.a` with `libACPIdentity_iOS.a`.
https://github.com/Adobe-Marketing-Cloud/acp-sdks/blob/master/iOS/ACPCore/libACPIdentity_iOS.a